lib/components/adapter.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @private
+ * @class Adapter
+ * @description Making requests to Shopify API.
+ * @requires request
+ * @requires bluebird
+ */
+'use strict';
+const request = require('request');
+const bluebird = require('bluebird');
```

Comment:
Yes, it's only there. I've been wondering between native promises or bluebird, but I chose bluebird because of it's performance, since this is a hot sport in the application.